### PR TITLE
chore: increase timeout for upgrade/upload tx types test

### LINKED ISSUE #3150: Increase tests' timeout to prevent flakiness
We need to increase the test timeout for the [Upgrade and Upload](https://github.com/FuelLabs/fuels-ts/blob/master/packages/fuel-gauge/src/transaction-upgrade.test.ts) transaction types to prevent test [flakiness](https://github.com/FuelLabs/fuels-ts/actions/runs/10815716921/job/30005255720?pr=3129#step:6:1012).

### PR DESCRIPTION
**Pull request by Torres-ssf**
- Closes #3150
- Closes TS-646

# Checklist

- [x] All **changes** are **covered** by **tests** (or not applicable)
- [x] All **changes** are **documented** (or not applicable)
- [x] I **reviewed** the **entire PR** myself (preferably, on GH UI)
- [x] I **described** all **Breaking Changes** (or there's none)

### CODE REVIEW

File: packages/fuel-gauge/src/transaction-upgrade.test.ts [hunk 1]
```diff
@@ -137,7 +137,7 @@ describe('Transaction upgrade state transition', () => {
     node.cleanup();
   });
 
-  it('should correctly upload bytecode in subsections slowly', { timeout: 38000 }, async () => {
+  it('should correctly upload bytecode in subsections slowly', { timeout: 76_000 }, async () => {
```
Comment (arboleya) [inline packages/fuel-gauge/src/transaction-upgrade.test.ts:140]: I believe @luisburigo initial tests worked with 60s here.

File: packages/fuel-gauge/src/transaction-upgrade.test.ts [hunk 2]
```diff
@@ -167,7 +167,7 @@ describe('Transaction upgrade state transition', () => {
     }
   });
 
-  it('should correctly upgrade chain with uploaded bytecode', { timeout: 15000 }, async () => {
+  it('should correctly upgrade chain with uploaded bytecode', { timeout: 40_000 }, async () => {
```
Comment (arboleya) [inline packages/fuel-gauge/src/transaction-upgrade.test.ts:170]: This I don't remember, did you try with 30s?

**Review by arboleya — DISMISSED** [review]

File: packages/fuel-gauge/src/transaction-upgrade.test.ts (hunk 2 above)
Comment (Torres-ssf) [reply packages/fuel-gauge/src/transaction-upgrade.test.ts:170]: @arboleya no. I've just use `40_000` to be sure

Comment (arboleya) [reply packages/fuel-gauge/src/transaction-upgrade.test.ts]: 🚢

**Review by maschad — APPROVED** [review]

**Review by Dhaiwat10 — APPROVED** [review]

**Pull request merged**